eclipse-collections/src/main/java/org/eclipse/collections/impl/utility/ArrayListIterate.java

```diff
@@ -109,7 +109,7 @@
                 data.setAccessible(true);
                 size.setAccessible(true);
             }
-            catch (SecurityException ignored)
+            catch (Exception exception)
```

Comment:
I agree, it looks awkward. I can pull the try-catch block up and just catch an Exception to ignore it in the outer block. Is this what you are suggesting?